### ISSUE TITLE
Can't get rid of ES-lint errors that state : JSX props should not use arrow functions  react/jsx-no-bind

### ISSUE THREAD

**Issue by davekaj** [PolymathNetwork/polymath-issuer#21]
Can't seem to get rid of this error no matter what I do, I tried to write /* eslint-disable jsx-no-bind */
 at the top of the page, but it didnt work (I tried multiple different syntaxes).

My theory, and I might be wrong, is that it is not part of the main eslint package, and therefore doesnt get picked up by eslint-disable. it is part of this plugin: https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md

As a solution, I recommend updating this line to read `true` https://github.com/PolymathNetwork/eslint-plugin-polymath-react/blob/0cd34e915433f8009cbba938446f586046e86997/config.js#L62

I think we could do without the rule , as I am just trying to disable it anyways.

@bshevchenko  let me know if we can update the linting rule to true

**Comment by Adam0000** [comment]
This should work for disabling: `/* eslint-disable react/jsx-no-bind */`

However, this rule is coming from [our React ESLint configuration](https://github.com/PolymathNetwork/eslint-plugin-polymath-react/blob/0cd34e915433f8009cbba938446f586046e86997/config.js#L57) so we should discuss it. I remember you showing me some places you were using arrow functions and it seemed like not using them would just make things messier. Do you have one or two examples like that?

**Comment by davekaj** [comment]
https://github.com/PolymathNetwork/polymath-issuer/blob/d41e2094be49c620e915222018df5f1613bf4e1a/src/app/dashboard/whitelist/WhiteListPage.js#L203-L221

Right here are the two main ones. selectedRows is a prop directly from the carbon component. I couldnt figure out how to write it any other way. 

Maybe there is an easy fix here , but without the arrow function I couldn't figure out how to pass selectedRows onwards

**Comment by Adam0000** [comment]
Below is an approach that would eliminate the error but it's less clean than yours. Maybe @bshevchenko has an idea?

```
 dataTableRender = ({ 
  rows, 
  headers, 
  getHeaderProps, 
  getSelectionProps, 
  getBatchActionProps, 
  onInputChange, 
  selectedRows, 
}) => {
  const removeInvestor = () => this.removeInvestor(selectedRows)
  const handleEditInvestors = () => this.handleEditInvestors(selectedRows)

  return (
    <TableContainer> 
      <TableToolbar> 
        <TableBatchActions {...getBatchActionProps()}> 
          <TableBatchAction onClick={removeInvestor}> 
              Remove Investor 
          </TableBatchAction> 
          <TableBatchAction onClick={handleEditInvestors}> 
              Edit Sale/Purchase Lockup Dates 
          </TableBatchAction> 
        </TableBatchActions> 
```

**Comment by bshevchenko** [comment]
From https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md
> A bind call or arrow function in a JSX prop will create a brand new function on every single render. This is bad for performance, as it will result in the garbage collector being invoked way more than is necessary. It may also cause unnecessary re-renders if a brand new function is passed as a prop to a component that uses reference equality check on the prop to determine if it should update.

So please don't try to deceive eslint, it won't solve the problem.
But since we need new function on every single render here (to pass new selectedRows value) `/* eslint-disable react/jsx-no-bind */` will be ok.

**Comment by davekaj** [comment]
yes this has been fixed in the 'whitelist' branch. i will close this issue once this branch gets merged into master
